android/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "TestApp",
```

Comment:
Do we need this file?

Reply:
Actually no. I just need to make sure that the path to node_modules folder points to the correct location. Might use Sam's suggestion for this.

Reply:
It's gone now.